src/core/friendly_errors/browser_errors.js

```diff
@@ -0,0 +1,20 @@
+// Browsers use error strings to build their error messages. These can be
+// different across different browsers. We can build a list of these strings
+// so that the FES is able to extract the required info from the errors given
+// by the browser
+const strings = {
+  ReferenceError: [
+    {
+      msg: '{{}} is not defined',
+      type: 'NOTDEFINED',
+      browser: 'all'
```

Comment:
why `all` instead of 'Chrome, Firefox, Edge and Opera' as in the comment?

Reply:
The idea was that there will be many browsers that use one particular string and a few that decide to go their own way by having their unique string.
But sometimes it can also happen that a browser will use the common string for some sub-cases of the same error, and use a unique string for other sub-cases.  For e.g. 
for [this error](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Errors/Cant_access_lexical_declaration_before_init), firefox has a different string, but it also uses `x is not defined` for other error-cases

So as the file grows over the next stages of the project, my idea was to have all browsers first try matching the error of `browser: all` and then if nothing matches there, it will try to look if there is a string marked with its name. The `all` here indicates that all browsers should try if this string matches. Maybe it could be different.

Reply:
Ah that makes sense!